packages/webchat/src/core/api.tsx

```diff
@@ -78,6 +63,7 @@ export default class WebchatApi {
     }
   }
 
+  // TODO: Fis this
   async resetSession(conversationId: uuid): Promise<void> {
     try {
       await this.axios.post('/conversations/reset', { ...this.baseUserPayload, conversationId }, this.axiosConfig)
```

Comment:
Maybe this should be considered debugger stuff? If a user wants a new conversation we can simply make an API call to create a new conversation. There is no need for a "reset conversation" concept

Reply:
Right! Not sure if this was the case, but maybe this code should have been removed when I deleted the emulator code?